doc/Makefile.am

```diff
@@ -2,8 +2,6 @@
 # Makefile.am for .../agent-ldap/doc
 #
 
-SUBDIRS = autodocs
```

Comment:
This ignores the `autodocs` directory but the content is still kept. Remove also the `Makefile.am` and `.gitignore` file there.